backend.native/tests/build.gradle

```diff
@@ -2612,6 +2612,9 @@ if (isMac()) {
                 args "-lobjc", '-fobjc-arc'
                 args '-fPIC', '-shared', '-o', "$buildDir/libobjcsmoke.dylib"
             }
+            if (project.testTarget == 'ios_x64') {
```

Comment:
What if `iphone_sim` is specified instead of `ios_x64`? May be it's better to create a `KonanTarget` instance from this property and check it instead of just string?